frontend/src/metabase-types/api/settings.ts

```diff
@@ -285,13 +239,62 @@ interface InstanceSettings {
   "show-static-embed-terms": boolean | null;
 }
 
+interface PublicSettings {
+  "anon-tracking-enabled": boolean;
+  "application-font": string;
+  "application-font-files": FontFile[] | null;
+  "application-name": string;
+  "available-fonts": string[];
+  "available-locales": LocaleData[] | null;
+  "cloud-gateway-ips": string[] | null;
+  "custom-formatting": FormattingSettings;
+  "custom-homepage": boolean;
+  "custom-homepage-dashboard": number | null;
+  "email-configured?": boolean;
+  "embedding-app-origin": string;
+  "enable-enhancements?": boolean;
+  "enable-password-login": boolean;
+  engines: Record<string, Engine>;
+  "ga-enabled": boolean;
+  "google-auth-client-id": string | null;
+  "google-auth-enabled": boolean;
+  "has-user-setup": boolean;
+  "hide-embed-branding?": boolean;
+  "help-link": HelpLinkSetting;
+  "help-link-custom-destination": string;
+  "is-hosted?": boolean;
+  "is-metabot-enabled": boolean;
+  "ldap-configured?": boolean;
+  "ldap-enabled": boolean;
+  "loading-message": LoadingMessage;
+  "map-tile-server-url": string;
+  "other-sso-enabled?": boolean | null;
```

Comment:
This attribute is not present in GET `/api/session/properties` response in my local instance.

Reply:
🤷‍♂️ 

https://github.com/metabase/metabase/blob/be-user-local-settings/enterprise/backend/src/metabase_enterprise/sso/integrations/sso_settings.clj#L255

Reply:
This seems to be an Enterprise setting. Probably that's why.
TBH, I don't know how to treat enterprise-only settings that are also public. Should they stay here, or should they be moved to the Enterprise type?

Reply:
@kamilmielnik moved it to EnterpriseSettings for now.

Reply:
> @kamilmielnik moved it to EnterpriseSettings for now.

Good idea :+1: 

> TBH, I don't know how to treat enterprise-only settings that are also public. Should they stay here, or should they be moved to the Enterprise type?

Or maybe we need `EnterprisePublicSettings`?

Reply:
I think we'll eventually end up having Settings as a union of AdminSettings | PublicSettings, etc.
And in the same way, we'll have EnterpriseSettings as a union of EnterpriseAdminSettings | EnterprisePublicSettings, etc.

Reply:
Unfortunately, had to move it back to unblock this PR.
https://github.com/metabase/metabase/pull/39468/commits/26d905a606c2f12c0f6265c8012d3fb953cbb512

The proper solution would be bigger than all changes in this PR so I'll make sure to follow up.
See [this Slack convo](https://metaboat.slack.com/archives/C0645JP1W81/p1709639951515289) for more context.